### ISSUE TITLE
Dark themes - keyframe arrows incorrectly display as disabled

### ISSUE THREAD

**Issue by artisteacher** [opentoonz/opentoonz#1176]
On all themes except for the 2 light themes (Gray 128 and Astral 128), the arrows that allow you to toggle through keyframes look disabled when they are functional. The actual disabled arrow buttons look like they should be functional. Hover states are working correctly. This is true for the keyframe arrow buttons in the viewer, function editor, and the level palette.

In the following screenshots, the last keyframe is selected so the right arrow should be disabled and the left arrow should be enabled:

In a light theme (correct)
![screen shot 2017-04-28 at 8 30 48 am](https://cloud.githubusercontent.com/assets/21159116/25528682/01588628-2bed-11e7-9cf3-15c59f592b0d.png)
In a dark theme (incorrect)
![screen shot 2017-04-28 at 8 25 46 am](https://cloud.githubusercontent.com/assets/21159116/25528642/c61b95d2-2bec-11e7-9e97-3423d8225289.png)

Edit: Unless I missed something in the qss, it looks like these arrows are hardcoded.

**Comment by turtletooth** [comment]
These are hardcoded into the UI, so I'm trying to figure out how to make them change based on theme.

**Comment by turtletooth** [comment]
@shun-iwasawa Do you have any ideas on how to move a QIcon to qss?  Or the other (possibly temporary solution) would be to just swap the active and inactive icons since the majority of themes are darkish.

**Comment by shun-iwasawa** [comment]
According to [here](https://forum.qt.io/topic/68151/change-qaction-icon-in-stylesheet), you can style the `QToolButton` that will be generated from `QAction`. 
I tried to set icons using `qproperty-icon` property in the qss but it failed for some unknown reason.
Finally I managed with a way as follows. It's dirty, but it works:

In `KeyframeNavigator::KeyframeNavigator()` :

```
. . . 
  QPixmap emptyPixmap(23, 23); // set transparent icon to show button's background
  emptyPixmap.fill(Qt::transparent);
  QIcon emptyIcon(emptyPixmap);

  m_actPreviewKey = new QAction(emptyIcon, tr("Previous Key"), this);
  connect(m_actPreviewKey, SIGNAL(triggered()), SLOT(togglePrevKeyAct()));
  addAction(m_actPreviewKey);
  QWidget* widget = widgetForAction(m_actPreviewKey); // obtain a widget generated from QAction
  widget->setObjectName("PreviousKey");
. . . 
```

And in the LESS file :

```
. . .
#keyFrameNavigator #PreviousKey{
	image: url("@{image_url}/prevkey.png");
	&:disabled{
		image: url("@{image_url}/prevkey_disabled.png");
	}
	// Followed by other properties, if needed
}
. . .
```

**Comment by turtletooth** [comment]
Fixed in #1244
